pkgs/applications/networking/modem-manager-gui/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, buildEnv, pkgconfig, python3, fetchhg, gtk3-x11, glib, gdbm, gtkspell3, itstool, libappindicator-gtk3, meson, ninja, perldevelPackages }:
+
+stdenv.mkDerivation rec {
+  name = "modem-manager-gui";
+  version = "0.4.17";
+  rev = "v${version}";
+
+  src = fetchhg {
+    url = https://linuxonly@bitbucket.org/linuxonly/modem-manager-gui;
+    rev = "8bcf842";
+    sha256 = "04p08gzvi068bd3slbnaghajwzp4h5jacib640k5shngqimkl84p";
+  };
+
+  buildInputs = [
+    pkgconfig
```

Comment:
Programs executed during build should go to `nativeBuildInputs`.

Reply:
fixed